src/grammar/next/grammar.gg

```diff
@@ -270,6 +271,7 @@ Parens = child:parens;
 
 StructInstance = type:TypeId "{" fields:commaList<StructFieldInitializer>? "}";
 InitOf = keyword<"initOf"> name:Id params:parameterList<expression>;
+CodeOf = keyword<"codeOf"> name:Id;
```

Comment:
this can break existing contracts

Reply:
![telegram-cloud-photo-size-2-5316612301027141746-y](https://github.com/user-attachments/assets/b6b91d1f-4252-4f8b-9317-d746abec3dbb)

Reply:
Do you have a query for closed-source contracts? e.g. PixelSwap? :)

Reply:
Fixed